bitnami/common/templates/_affinities.tpl

```diff
@@ -47,11 +47,10 @@ Return a nodeAffinity definition
 
 {{/*
 Return a topologyKey definition
-{{ include "common.affinities.topologyKey" (dict "topologyKey" "BAR") -}}
+{{ include "common.affinities.topologyKey" (dict "topologyKey" .topologyKey) -}}
```

Comment:
Sorry, I missed it. At this point, I think we should leave "BAR" var because this is an example parameter and we should not reference a value parameter.

Reply:
fixed